mkdata.py

```diff
@@ -9,27 +10,54 @@
 Licensed under the BSD 3-clause license.
 """
 
+try:
+    import click
+except ImportError:
+    from mock import Mock
+    click = Mock()
+    def call(*args, **kargs):
+        def decorator(f):
+            return f
+        return decorator
+    click.command = call
+    click.argument = call
+    click.option = call
 import os
 import struct
 
-# how often do you want the test data to be duplicated?
-# it won't be exact duplicates to avoid deduplication.
-N = 5000
-
 # each BLK_SIZE bytes (or, if file is smaller, in the middle of the file),
 # we insert a counter into the output file to avoid creating a duplicate.
 BLK_SIZE = 65536
 
-# arrange some test src data in this directory
-# make sure it fits into RAM as it will all be read into RAM, except the
-# "sparse" file. you may put 1 sparse file in there, name it "sparse".
+N = 5000
 SRC = 'testdata'
-
-# where we put the output data, make sure there is enough space.
 DST = 'lotsofspace'
 
+@click.command()
+@click.argument('src', default=SRC)
+@click.argument('dst', default=DST)
+@click.option('--times', default=N,
```

Comment:
also maybe no default here, it depends on the volume in the testdata the user has and how much he wants.
there's no reasonable default for this.